### PR TITLE
Update docker tag in readme

### PR DESCRIPTION
**Pull request by srknzl**
because it is outdated

### CODE REVIEW

**Review by yuce — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -44,9 +44,12 @@ The quickest way to start a single member cluster for development purposes is to
 [Docker images](https://hub.docker.com/r/hazelcast/hazelcast/).
 
 ```bash
-docker run -p 5701:5701 hazelcast/hazelcast:4.1.1
+docker run -p 5701:5701 hazelcast/hazelcast:latest
```
Comment (mdumandag) [inline README.md:47]: Is the `latest` tag necessary here? I thought it was the default value when you don't specify a tag. Maybe we can remove it

Comment (srknzl) [reply README.md]: done in 0e438611

**Comment by srknzl** [comment]
@yuce pls rereview, I deleted latest tag

File: README.md [hunk 2]
```diff
@@ -44,9 +44,12 @@ The quickest way to start a single member cluster for development purposes is to
 [Docker images](https://hub.docker.com/r/hazelcast/hazelcast/).
 
 ```bash
-docker run -p 5701:5701 hazelcast/hazelcast:4.1.1
+docker run -p 5701:5701 hazelcast/hazelcast
 ```
 
+This command uses the `latest` tag in order to fetch the latest hazelcast version. You can find all available tags
```
Comment (mdumandag) [inline README.md:50]:
```suggestion
This command fetches the latest Hazelcast version. You can find all available tags
```

Comment (srknzl) [reply README.md:50]: Is it always the case? What is latest tag in hazelcast docker? Is it master snapshot?  Is it 5.0? is it 5.1-beta? or 5.1? It is updated 2 months ago. We may lead users to wrong version.

Comment (srknzl) [reply README.md:50]: Maybe just saying `latest available hazelcast docker version` is better

Comment (mdumandag) [reply README.md:50]: Not a snapshot, it is just the most recently released patch version of the latest minor version. So, it should be okay to say that it is the latest Hazelcast version IMHO

Comment (srknzl) [reply README.md:50]: ok changing it then. I think they did not update docker hub yet after 5.1 release.

File: README.md [hunk 3]
```diff
@@ -44,9 +44,12 @@ The quickest way to start a single member cluster for development purposes is to
 [Docker images](https://hub.docker.com/r/hazelcast/hazelcast/).
 
 ```bash
-docker run -p 5701:5701 hazelcast/hazelcast:4.1.1
+docker run -p 5701:5701 hazelcast/hazelcast
 ```
 
+This command fetches the latest Hazelcast version. You can find all available tags
+[here](https://hub.docker.com/r/hazelcast/hazelcast/tags).
+
```
Comment (mdumandag) [inline README.md:52]: BTW, I just saw that there is a link at the top, to a similar page. Maybe this sentence is not required at all

Comment (srknzl) [reply README.md:52]: I just wanted to emphasize the way they can use to use other hazelcast versions. It is also a good practice to do so in Dockerfile.

**Pull request merged**